.tekton/tasks/task-switchboard.yaml

```diff
@@ -53,3 +56,6 @@ spec:
         'data[_]' \
         | jq '[.result.[].expressions.[].value | to_entries | .[] | select(.value == true) | .key]' \
         | tee "$(results.bindings.path)"
+
+        # Output is "execute_e2e" or "dont_execute_e2e" based on files changed in pr
```

Comment:
Technically the output isn't `dont_execute_e2e` but `No need to run e2e tests\ndont_execute_e2e`, because the script prints everything to stdout. It might be good to clean that up - always print to stderr except for the actual output of the script